cmake/RunTests.cmake

```diff
@@ -34,6 +37,9 @@ execute_process(
   RESULT_VARIABLE res
   ${EXTRA_ARGS})
 
+file(REMOVE ${WORKING_DIR}/Xtest_rplugin_manifest)
+file(REMOVE_RECURSE ${WORKING_DIR}/Xtest_xdg)
```

Comment:
I had a semi hectic day and think I didn't understand the question earlier.  If you're asking if the files are "deleted" before the tests run, from what I tried they seem to be evaluated sequentially and after the tests run.

My ghetto confirmation of this was running `watch -d -n 0.1 'ls -ald X*'` in the repo directory while tests ran.